cpr/session.cpp

```diff
@@ -381,8 +381,6 @@ Response Session::Impl::makeRequest(CURL* curl) {
     auto protocol = url_.substr(0, url_.find(':'));
     if (proxies_.has(protocol)) {
         curl_easy_setopt(curl, CURLOPT_PROXY, proxies_[protocol].data());
-    } else {
-        curl_easy_setopt(curl, CURLOPT_PROXY, "");
```

Comment:
Seems like it might be better to set this to nullptr (which I think is different from the empty string) instead of not setting it at all.  I would fear the handle would be reused after the cpr proxies had been cleared and the change not reflected.